packages/legend-studio/src/models/metamodels/pure/model/packageableElements/domain/PropertyReference.ts

```diff
@@ -69,7 +69,9 @@ export abstract class PropertyReference
     return [
       CORE_HASH_STRUCTURE.PROPERTY_POINTER,
       this.value.name,
-      this.ownerReference.valueForSerialization,
+      this.value.owner instanceof Association
```

Comment:
You're changing hash computation on `metamodel` side, where is the `protocol` counterpart?